.clang-format

```diff
@@ -1,3 +1,5 @@
+## I'm a little teapot, short and stout.
+
```

Comment:
needs a fix, not elegant

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Elegant